modules/freetype/README.md

```diff
@@ -1,34 +1,33 @@
-FreeType Module
-===========
+FreeType2 Wrapper Module
+==========================
 
-This FreeType module allows you to draw strings with outlines and bitmaps.
+This FreeType2 wrapper module allows to draw strings with outlines and bitmaps.
 
-Installation
------------
+Requested external libraries.
+------------------------------
```

Comment:
Please remove dots from the `Headers`

Please align number of `===` / `---` symbols on the next line (as markdown requested)

Reply:
Thank you very much ! I'll fix it.